app/src/main/java/com/example/marvelcomicsapp/ui/searchcomics/SearchComicsScreen.kt

```diff
@@ -204,9 +205,9 @@ fun TransparentHintTextField(
     ) {
         val searchColor = {
             if (isHintVisible) {
-                Color.LightGray
+                Gray300
             } else {
-                Color.Gray
+                Gray400
```

Comment:
In that shorts examples, you can type just:
if (isHintVisible) Gray300 else Gray400